tests/integration/test_vulnerability_detector/test_providers/test_providers_no_os.py

```diff
@@ -56,17 +109,36 @@ def get_configuration(request):
 
 @pytest.mark.skip(reason="It will be blocked by #1602, when it was solve we can enable again this test")
 def test_providers_no_os(clean_vuln_tables, get_configuration, configure_environment):
-    """Check if modulesd downloads the feeds without specifing the os version.
-
-    For this purpose, on the one hand, it checks if errors occur when the <os> tag
-    is omitted in the configuration in providers that should have it and, on the other hand,
-    if the update of the feeds starts normally in providers that do not require this tag.
-
-    Args:
-        clean_vuln_tables: Clean the tables involved with vulnerability detector packages and feeds.
-        get_configuration (fixture): Get configurations from the module.
-        configure_environment (fixture): Configure a custom environment for testing.
-    """
+    '''
+    description: Check if modulesd downloads the feeds without specifing the os version. To do this, it checks if errors
+                 occur when the <os> tag is omitted in the configuration in providers that should have it and, on the
+                 other hand, if the update of the feeds starts normally in providers that do not require this tag.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean the tables involved with vulnerability detector packages and feeds.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+
+    assertions:
+        - Verify that the provider os data update starts when `os` has not a determined value.
+        - Verify that an error message appears when using `error` tag in providers.
+
+    input_description:
+        - Test cases are found in the test module and include parameters for the `os` tag of the provider option and
+          `error` tag. The `wazuh_providers_os.yaml` file provides the configuration of this module for this test.
+
+    expected_output:
+        - 'Error log Configuration error at /var/ossec/etc/ossec.conf not found'
+        - r'Could not find .* update starting log'
```

Comment:
Add the following callbacks

```suggestion
        - r'.*: Configuration error at.*'
        - r'Starting .* database update'
```